src/couch_replicator/src/couch_replicator_job_sup.erl

```diff
@@ -1,34 +0,0 @@
-% Licensed under the Apache License, Version 2.0 (the "License"); you may not
```

Comment:
This is a left-over supervisor from many years ago when we switched to the scheduling replicator and forgot to remove it. Since we're cleaning up supervisors, removing the extra junk as well.